packages/tasks/src/local-apps.ts

```diff
@@ -39,36 +45,52 @@ export type LocalApp = {
 			 * And if not (mostly llama.cpp), snippet to copy/paste in your terminal
 			 * Support the placeholder {{GGUF_FILE}} that will be replaced by the gguf file path or the list of available files.
 			 */
-			snippet: (model: ModelData) => string | string[];
+			snippet: (model: ModelData) => Snippet | Snippet[];
 	  }
 );
 
 function isGgufModel(model: ModelData) {
 	return model.tags.includes("gguf");
 }
 
-const snippetLlamacpp = (model: ModelData): string[] => {
+const snippetLlamacpp = (model: ModelData): Snippet[] => {
+	const command = (binary: string) =>
+		[
+			"# Load and run the model:",
+			`${binary} \\`,
+			'  --hf-repo "${model.id}" \\',
+			"  --hf-file {{GGUF_FILE}} \\",
+			'  -p "You are a helpful assistant" \\',
+			"  --conversation",
```

Comment:
I fixed an issue related to this yesterday. For now, when a chat template is not compatible, llama.cpp will default to using chatml (https://github.com/ggerganov/llama.cpp/pull/8160). That should prevent crash the app, but I agree that it's not idea to just assume chatml for all non-supported models.

Because all other local apps open in conversation mode by default, so I think it would be nice if llama.cpp is defaulted to `--conversation` mode like the other.

Reply:
Good call. Yeah! I'm good with this. Resolving this comment!